habitat-baselines/habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -398,11 +282,11 @@ def save_checkpoint(
             None
         """
         checkpoint = {
-            "state_dict": self.agent.state_dict(),
+            **self._agent.get_save_state(),
             "config": self.config,
         }
         if extra_state is not None:
-            checkpoint["extra_state"] = extra_state
+            checkpoint["extra_state"] = extra_state  # type: ignore
```

Comment:
Why the added `# type: ignore` did the code change or was this broken to begin with ?